doc/explanations/faq.rst

```diff
@@ -1,218 +1,337 @@
-********************************************************************************
+**Q. What is a one-time minting policy?**
 
-In a one-time minting policy, the complete set of tokens scoped under it is minted by one specific transaction. This means that no more tokens will ever be minted under that policy. This type of policy does require smart contracts and cannot be expressed using multisig.
+A. In a one-time minting policy, the complete set of tokens scoped under
+it is minted by one specific transaction. This means that no more tokens
+will ever be minted under that policy. This type of policy does require
+smart contracts and cannot be expressed using multisig.
 
-A use case of a one-time minting policy would be minting ticket tokens for a specific gig. The venue capacity is known ahead of time, so there’ll be no need to ever allow more tickets to be minted.
+A use case of a one-time minting policy would be minting ticket tokens
+for a specific gig. The venue capacity is known ahead of time, so
+there’ll be no need to ever allow more tickets to be minted.
 
 Multi-asset structure, representation and properties
-#########################################################
+====================================================
 
-What is fungibility and non-fungibility?
-********************************************************************************
+**Q. What is fungibility and non-fungibility?**
 
-Fungibility is a relation between two assets/tokens. Tokens are said to be fungible with each other when they are interchangeable. For example, fiat money is fungible as a $10 bill is interchangeable with all other (real) $10 bills (and all 10-sets of $1 bills, and all pairs of $5s).
+A. Fungibility is a relation between two assets/tokens. Tokens are said
+to be fungible with each other when they are interchangeable. For
+example, fiat money is fungible as a $10 bill is interchangeable with
+all other (real) $10 bills (and all 10-sets of $1 bills, and all pairs
+of $5s).
 
-Non-fungible assets are not interchangeable with each other. For example, two diamonds, or two on-chain tokens representing the two real-world diamonds. If there are no other assets a token is fungible with -such as a token representing a house- the token is deemed to be unique (non-fungible).
+Non-fungible assets are not interchangeable with each other. For
+example, two diamonds, or two on-chain tokens representing the two
+real-world diamonds. If there are no other assets a token is fungible
+with -such as a token representing a house- the token is deemed to be
+unique (non-fungible).
 
-What is a token bundle?
-********************************************************************************
-
-A mixed collection of tokens scoped under one or more minting policies. Any tokens can be bundled together.
+**Q. What is a token bundle?** A. A mixed collection of tokens scoped
+under one or more minting policies. Any tokens can be bundled together.
```

Comment:
do you want to mention asset IDs?

Reply:
I think we have a nice explainer in the dedicated section (assets, tokens, and token bundles). No need to write too much here.